### ISSUE TITLE
Draft is_deleted missing

### ISSUE THREAD

**Issue by fenekku** [inveniosoftware/invenio-drafts-resources#62]
Package version (if known): 0.4x and invenio-records-resources 0.6.x

## Describe the bug
Had to skip : 

```python
@pytest.mark.skip()  # WHY: Draft::is_deleted is not implemented
def test_mutiple_edit(client, identity_simple, input_data):
```

because `is_deleted` disappeared at some point from the `Draft` object either from records-resources or drafts-resources.

Someone more familiar (hint hint @ppanero ) would be able to tell where it disappeared.

**Comment by lnielsen** [comment]
It lives in Invenio-Records, hence it might have been an outdated dependency?

**Comment by ppanero** [comment]
@lnielsen yes, indeed.

Tests pass on my local (since it installs latest). `is_deleted` is implemented on invenio-records v1.4.0a4. So if you have an older dep, it will fail. I am re-enabling the tests (pinning that dep in records-resources <https://github.com/inveniosoftware/invenio-records-resources/pull/124>) so we have a more thorough test suit. I could understand if they failed in lowest,  was travis also failing in release/devel mode?. Travis works in mysterious ways I guess...

Tests are fully passing on travis too:  https://travis-ci.org/github/inveniosoftware/invenio-drafts-resources/builds/732883692
